testdata/provenances/15dc16c42a4ac9ed77f337a4a3065a63e444c29c18c8cf69d6a6b4ae678dca5c.json

```diff
@@ -1,52 +0,0 @@
-{
```

Comment:
Why is this deleted? I think it is good to have an example SLSA provenance file. I think it is better to update it according to our new format, and add logic that verifies it.